answer/166ryoryo/HTML/02/anchor.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta charset="utf-8">
+<title>anchor</title>
+<meta name="keywords" content="">
+<meta name="description" content="">
+</head>
+
+<body>
+  <a href="https://isc-makeit.github.io/Make-IT-homepage/">Make-IT</a>
+  <a href="https://isc-makeit.github.io/Make-IT-homepage/" target="_blank">make</a>
+  <script type="text/javascript">
+    function openWin() {
```

Comment:
```suggestion
    function openWin() {
        window.open(
            "https://isc-makeit.github.io/Make-IT-homepage/",
            "_blank",
            "menubar=0,
            width=300,
            height=200,
            top=200,
            left=200"
        );
    }
```
インデント気をつけましょう 👀 
カンマで改行するなら全部改行した方が統一感がありそうですね